src/main/resources/reference.conf

```diff
@@ -253,6 +253,18 @@ resourceTypes = {
       stop_start_notebook_cluster = {
         description = "stop and start notebook clusters in this billing-project"
       }
+      list_persistent_disk = {
+        description = "list all persistent disks in this billing-project"
+      }
+      create_persistent_disk = {
+        description = "create persistent disk in this billing-project"
+      }
+      modify_persistent_disk = {
```

Comment:
I'm not sure we need "modify" at the project level -- I don't see project owners with the ability to modify (e.g. resize) all disks in the project. Just list/create/delete seems sufficient.